planter/planter.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: use like: ./planter.sh bazel build //cmd/...
+# NOTE: build kubernetes with: <path to test-infra>/planter.sh make bazel-build
+
+set -o errexit -o nounset
+IMAGE_NAME="gcr.io/k8s-testimages/planter"
+VERSION="0.1"
+IMAGE="${IMAGE_NAME}:${VERSION}"
+# run our docker image as the host user with bazel cache and current dir
+VOLUMES="-v ${DIR}:${DIR} -v ${HOME}:${HOME} --tmpfs /tmp:exec,mode=777"
```

Comment:
$DIR no longer exists?

Reply:
Agh, yes. why did this not fail? :O

Reply:
Oh. I suspect you can only `set -o` one thing per line. so the `-o unset` was not being applied.

Reply:
done, and fixed errorexit / nounset.